### PR TITLE
Issue11572fix old

### PR DESCRIPTION
**Pull request by TomasZilinek**
I replaced all calls to the old function by calls to the new. I think it is not too much. :D

### CODE REVIEW

File: src/openrct2/localisation/Language.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*****************************************************************************
- * Copyright (c) 2014-2020 OpenRCT2 developers
+ * Copyright (c) 2014-2019 OpenRCT2 developers
```
Comment (janisozaur) [inline src/openrct2/localisation/Language.h:2]: moved back in time?

Comment (TomasZilinek) [reply src/openrct2/localisation/Language.h:2]: I have no idea what happened. I merged OpenRCT2 with my fork, pulled and this happened.

Comment (Gymnasiast) [reply src/openrct2/localisation/Language.h:2]: You need to rebase to the latest `develop`. Do not merge our branches back into yours.

**Comment by tupaschoal** [comment]
Also, please rebase the branch onto `develop`, squash the commits and reword it like "Fix #11572: create gfx_draw_string() overload"

**Comment by TomasZilinek** [comment]
> Also, please rebase the branch onto `develop`, squash the commits and reword it like "Fix #11572: create gfx_draw_string() overload"

Hi I am going to close this PR and create another one after I sort everything out at my side.

**Comment by Gymnasiast** [comment]
> Hi I am going to close this PR and create another one after I sort everything out at my side.

Please, NEVER do that. It just makes earlier discussion more difficult to find and causes unnecessary notifications. Just fix the existing PR. You can mark your PR as "Draft" if you need more time to make changes.

**Comment by TomasZilinek** [comment]
> > Hi I am going to close this PR and create another one after I sort everything out at my side.
> 
> Please, NEVER do that. It just makes earlier discussion more difficult to find and causes unnecessary notifications. Just fix the existing PR. You can mark your PR as "Draft" if you need more time to make changes.

Can I never do that from now on? Because I am unable to rebase now. (and it would have been very difficult before). Now finally I can at least continue working on this.

**Comment by Gymnasiast** [comment]
Yes, that's fine. Just wanted to make sure you knew.

As long as you make sure to create a new branch when you start work on something, Git provides all the tools you need to correct errors. If needed, we can help you.

**Comment by TomasZilinek** [comment]
Yeah I did not want to annoy you with things I could probably search on Google. But after some hours of failure to get my fork to the previous valid state I decided to delete it and copy the correct files (with work already done) to a local clone of the new fork.

Speaking about google. I don't know if it was caused by a software update on my machine or it was my mistake, but I cannot execute "make" to compile OpenRCT2 (It compiled earlier just fine). There are some problems with google benchmarks. Can you help me with that?

This is output of "cmake .." executed from build directory on ubuntu 18.04:

-- CMAKE_BUILD_TYPE not explicitly set. Not enabling IPO.
No discord-rpc detected, to enable clone discord-rpc to root directory: /home/thomasz/school/semester4/pb173oss/ossProject/OpenRCT2
-- Unsupported CXXFLAG: -Wmissing-variable-declarations
-- Unsupported CXXFLAG: -Wunreachable-code-break
-- Unsupported CXXFLAG: -Wrange-loop-analysis
-- Unsupported CXXFLAG: -Wtautological-unsigned-zero-compare
CMake Warning at src/openrct2/CMakeLists.txt:92 (find_package):
  By not providing "Findbenchmark.cmake" in CMAKE_MODULE_PATH this project
  has asked CMake to find a package configuration file provided by
  "benchmark", but CMake did not find one.

  Could not find a package configuration file provided by "benchmark" with
  any of the following names:

    benchmarkConfig.cmake
    benchmark-config.cmake

  Add the installation prefix of "benchmark" to CMAKE_PREFIX_PATH or set
  "benchmark_DIR" to a directory containing one of the above files.  If
  "benchmark" provides a separate development package or SDK, be sure it has
  been installed.
Call Stack (most recent call first):
  CMakeLists.txt:267 (include)


Google benchmark not found, disabling support
-- Checking for module 'duktape'
--   No package 'duktape' found
CMake Error at /usr/share/cmake-3.10/Modules/FindPkgConfig.cmake:415 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.10/Modules/FindPkgConfig.cmake:593 (_pkg_check_modules_internal)
  src/openrct2/CMakeLists.txt:107 (PKG_CHECK_MODULES)
  CMakeLists.txt:267 (include)


-- Configuring incomplete, errors occurred!
See also "/home/thomasz/school/semester4/pb173oss/ossProject/OpenRCT2/build/CMakeFiles/CMakeOutput.log".
See also "/home/thomasz/school/semester4/pb173oss/ossProject/OpenRCT2/build/CMakeFiles/CMakeError.log".

**Comment by Gymnasiast** [comment]
```
-- No package 'duktape' found
CMake Error at /usr/share/cmake-3.10/Modules/FindPkgConfig.cmake:415 (message):
A required package was not found
```
This is your clue. You're missing a package. I think you should be able to guess which one. ;-)
Looks for it in your distro's repositories.

**Pull request closed without merging**